### DISCUSSION TITLE
windows10 搜索无效其他功能有用

### DISCUSSION THREAD

**Discussion by ghost** [1715173329/UnblockNeteaseMusic#482] [General]
![image](https://user-images.githubusercontent.com/98085002/150333985-6744938e-b84a-4f5c-a60d-bdc1f82aff34.png)

**Comment by 1715173329** [comment]
Duplicate of #480

**Reply by ghost** [reply]
可以详细说一下吗，没明白

**Reply by pan93412** [reply]
你發了兩次
